gradle/spotbugs-exclude.xml

```diff
@@ -28,8 +28,19 @@ For a detailed description of spotbugs bug categories, see https://spotbugs.read
         <!-- Disable warnings about mutable objects and the use of public fields.
             EI_EXPOSE_REP: May expose internal representation by returning reference to mutable object
             EI_EXPOSE_REP2: May expose internal representation by incorporating reference to mutable object
+            MS_EXPOSE_REP: Public static method may expose internal representation by returning array
```

Comment:
Why do we have to disable this one?

Reply:
1. Public static `org.apache.kafka.server.metrics.KafkaYammerMetrics.defaultRegistry()` may expose internal representation by returning `KafkaYammerMetrics.metricsRegistry`.
2. Public static `org.apache.kafka.common.security.auth.SecurityProtocol.names()` may expose internal representation by returning `SecurityProtocol.NAMES`